specs/experimental/fault-proof/index.md

```diff
@@ -156,7 +159,24 @@ Key: `5 ++ keccak256(commitment ++ z)[1:]`, where:
 - `commitment` is a bytes48, representing the KZG commitment.
 - `z` is a big-endian `uint256`
 
-#### Type `6-128`: reserved range
+#### Type `6`: Global EIP-4844 Point-evaluation precompile key
+
```

Comment:
Why is this necessary? We stub out the execution of the precompile in the program with a lookup to the oracle?

Reply:
Would love this kind of context to be included in the specs

Reply:
Good suggestion, thanks. I've added a section detailing this.